FlowCrypt/Controllers/Inbox/InboxViewController.swift

```diff
@@ -92,6 +92,7 @@ final class InboxViewController: ASDKViewController<ASDisplayNode> {
 extension InboxViewController {
     private func setupUI() {
         title = inboxTitle
+        navigationItem.setAccessibility(id: inboxTitle)
```

Comment:
Let's do `id: "navigationItem \(inboxTitle)"` to distinguish it from just the title